smartapps/plaidsystems/spruce-scheduler.src/spruce-scheduler.groovy

```diff
@@ -2056,6 +2035,8 @@ def getRainToday() {
 boolean isWeather(){
     if (!settings.isRain && !settings.isSeason) return false
     String wzipcode = zipString()
+    log.debug wzipcode
+    //log.debug "${getTwcLocation("${wzipcode}")}"
```

Comment:
And, of course, as above, you'll have to check the return of zipString incase someone is using ST without a location.

Reply:
Check getTwcLocation(wzipcode)?.location?.city and check if getTwcForecast and getTwcConditions return otherwise return false and ignore isWeather()